src/plugins/intel_cpu/src/nodes/eltwise.h

```diff
@@ -187,8 +187,10 @@ class Eltwise : public Node {
     void appendMemory(const std::vector<float> &data, MemoryPtr &memPtr, std::vector<const void*>& postOpsMem);
 
     bool canUseAclExecutor = false;
+    bool canUseShlExecutor = false;
     EltwiseAttrs eltwiseAttrs;
     std::shared_ptr<EltwiseExecutor> aclExecPtr = nullptr;
+    std::shared_ptr<EltwiseExecutor> shlExecPtr = nullptr;
```

Comment:
I have tried unified these two executors into one before, but I forgot the reasons why I gave up in the end. (It has been for a while) :(
Anyway, I will try that again! And if it can be done, that will be so great; if not, I will give you the detailed reasons. :)